src/Bling.js

```diff
@@ -406,26 +416,18 @@ class Bling extends Component {
         const reRenderProps = filterProps(Bling.reRenderProps, this.props, nextProps);
         const shouldRender = !propsEqual(reRenderProps.props, reRenderProps.nextProps);
         const shouldRefresh = !shouldRender && !propsEqual(refreshableProps.props, refreshableProps.nextProps);
-        // console.log(`shouldRefresh: ${shouldRefresh}, shouldRender: ${shouldRender}, isScriptLoaded: ${isScriptLoaded}, syncCorrelator: ${Bling._adManager._syncCorrelator}`);
+        // console.log(`shouldRefresh: ${shouldRefresh}, shouldRender: ${shouldRender}, isScriptLoaded: ${isScriptLoaded}`);
 
         if (shouldRefresh) {
             this.configureSlot(this._adSlot, nextProps);
         }
 
-        if (Bling._adManager._syncCorrelator) {
-            if (shouldRefresh) {
-                Bling._adManager.refresh();
-            } else if (shouldRender || isScriptLoaded) {
-                Bling._adManager.renderAll();
-            }
-        } else {
-            if (shouldRefresh) {
-                this.refresh();
-                return false;
-            }
-            if (shouldRender || isScriptLoaded) {
-                return true;
-            }
+        if (shouldRefresh) {
```

Comment:
Should you move this up to line 421? You already have a `shouldRefresh()` conditional